src/material/core/private/ripple-loader.ts

```diff
@@ -109,12 +109,6 @@ export class MatRippleLoader implements OnDestroy {
     }
   }
 
-  /** Returns the ripple instance for the given host element. */
-  getRipple(host: HTMLElement): MatRipple | undefined {
```

Comment:
Note: there's some more cleanup to be done in this file in order to stop instantiating `MatRipple` manually. I'll do it in a follow-up, because it'll conflict with #29620.